File: modul-06/python/PRAK603-2310817310003-Mfaridsetiawan.py
banyak_baris = 2
banyak_angka_baris, banyak_angka_baris1 = map(int, input().split())
if banyak_angka_baris == banyak_angka_baris1:
    wadah_array = []
    for i in range(banyak_baris):
        wadah_array.append(list(map(int, input().split())))
    for i in range(banyak_baris - 1):
        for j in range(banyak_angka_baris):
            if i + 1 < banyak_baris:
                wadah_array[i][j] = wadah_array[i][j] * wadah_array[i + 1][j]
    for i in range(banyak_baris - 1):
        for j in range(banyak_angka_baris):
            print(wadah_array[i][j], end=" ")
else:
    print("Jumlah Tidak Sama!")

File: modul-06/bahasa c/PRAK603-2310817310003-Mfaridsetiawan.c
#include <stdio.h>

int main()
{

    int banyak_baris, banyak_angka_baris, banyak_angka_baris1;
    banyak_baris = 2;
    scanf("%d %d", &banyak_angka_baris, &banyak_angka_baris1);
    if (banyak_angka_baris == banyak_angka_baris1)
    {
        int wadah_array[banyak_baris][banyak_angka_baris];

        for (int i = 0; i < banyak_baris; i++)
        {
            for (int j = 0; j < banyak_angka_baris; j++)
            {
                scanf("%d", &wadah_array[i][j]);
            }
        }

        for (int i = 0; i < banyak_baris - 1; i++)
        {
            for (int j = 0; j < banyak_angka_baris; j++)
            {
                if (i + 1 < banyak_baris)
                {
                    wadah_array[i][j] = wadah_array[i][j] * wadah_array[i + 1][j];
                }
            }
        }
        for (int i = 0; i < banyak_baris - 1; i++)
        {
            for (int j = 0; j < banyak_angka_baris; j++)
            {
                printf("%d ", wadah_array[i][j]);
            }
        }
    }
    else
    {
        printf("Jumlah Tidak Sama!");
    }
    return 0;
}